Resources/Scripts/1.1/Set-ImmutableIdForActiveDirectoryUsers.ps1

```diff
@@ -79,21 +79,9 @@ function Set-ImmutableIdForActiveDirectoryUsers {
 
 function Connect-AzureActiveDirectory {
     Try {
-        $requiredAzureModule = "Microsoft.Graph.Users"
-        if (-not (Get-Module -ListAvailable -Name $requiredAzureModule)) {
-            Write-Error "Missing required Azure module '$requiredAzureModule'"
-            Stop-ScriptExecution
-        }
-
-        # default connection name
-        $connectionName = "AzureRunAsConnection"
-
-        $servicePrincipalConnection = Get-AutomationConnection –Name $connectionName  
-
-        write-output "Logging in to Azure AD"
-        Connect-MgGraph -ClientID $servicePrincipalConnection.ApplicationId `
-                -TenantId $servicePrincipalConnection.TenantId `
-                -CertificateThumbprint $servicePrincipalConnection.CertificateThumbprint
+        Connect-AzAccount -Identity
```

Comment:
Only This part has been added to 1.2. I will revert this 1.1 change before merging.